### PR TITLE
orient detail pages based on components

### PR DESCRIPTION
**Pull request by kenjinp**
Added CreatedAt and CreatedBy to details page

Orients details page flex direction based on component type

### CODE REVIEW

File: apps/search-poc/src/utils/__tests__/index.test.ts [hunk 1]
```diff
@@ -1,140 +1,150 @@
-import { getLabel, parseNexusUrl, camelCaseToLabelString } from '../index';
```
Comment (kenjinp) [inline apps/search-poc/src/utils/__tests__/index.test.ts]: I just added the CamelToKebab test, prettier did some crazy stuff to this file

File: apps/search-poc/src/utils/__tests__/index.test.ts [hunk 2]
```diff
@@ -1,140 +1,150 @@
...
+  parseNexusUrl,
+  camelCaseToLabelString,
+  camelToKebab
+} from "../index";
 
-describe('utils', () => {
-  describe('camelCaseToLabelString', () => {
-    it('should convert camel cased string to capitalized words', () => {
-      expect(camelCaseToLabelString('camelCase')).toEqual('Camel Case');
+describe("utils", () => {
```
Comment (julienmachon) [inline apps/search-poc/src/utils/__tests__/index.test.ts]: Why did we go from single quotes to double quotes? 🤔

Comment (kenjinp) [reply apps/search-poc/src/utils/__tests__/index.test.ts]: My prettier had a stroke! 🙈

File: apps/search-poc/src/containers/ResourceDetails/ResourceDetails.tsx [hunk 3]
```diff
@@ -54,13 +59,18 @@ const ResourceDetailsContainer: React.FunctionComponent<{
   return (
     <ResourceDetails
       id={id}
+      className={Object.keys(components)
+        .map(componentName => `-${camelToKebab(componentName)}`)
```
Comment (julienmachon) [inline apps/search-poc/src/containers/ResourceDetails/ResourceDetails.tsx]: Why does it start with a hyphen?

Comment (kenjinp) [reply apps/search-poc/src/containers/ResourceDetails/ResourceDetails.tsx]: it's a variant css class 
https://rscss.io/variants.html

Comment (julienmachon) [reply apps/search-poc/src/containers/ResourceDetails/ResourceDetails.tsx]: wow

**Pull request merged**